client/helpers/index.js

```diff
@@ -145,3 +145,42 @@ export const stripInvalidXmlChars = text => {
   }
   return text.replace(/([^\x09\x0A\x0D\x20-\uD7FF\uE000-\uFFFC\u{10000}-\u{10FFFF}])/ug, '')
 };
+
+// protocol 1, protocol 2 and protocol 3
```

Comment:
Not sure what this comment is for, but it seems to represent neither the arguments nor return value of this function.

Reply:
it's _one_ of the return values - I can expand it a bit

Reply:
The whole map just results in `1, 2 and 3` though doesn't it? I don't see where "protocol" would be added.

Reply:
yeah that's what I meant, derp - I'll fix the comment